www/core/sts/list_sts.xml

```diff
@@ -13,14 +13,15 @@
-	<extension name="Joomla" element="joomla" type="file" version="3.10.12" targetplatformversion="3.10" detailsurl="https://update.joomla.org/core/extension.xml" />
-	<extension name="Joomla" element="joomla" type="file" version="4.3.4" targetplatformversion="4.0.4" detailsurl="https://update.joomla.org/core/sts/extension_sts.xml" />
-	<extension name="Joomla" element="joomla" type="file" version="4.3.4" targetplatformversion="4.0.5" detailsurl="https://update.joomla.org/core/sts/extension_sts.xml" />
-	<extension name="Joomla" element="joomla" type="file" version="4.3.4" targetplatformversion="4.0.6" detailsurl="https://update.joomla.org/core/sts/extension_sts.xml" />
+	<extension name="Joomla" element="joomla" type="file" version="4.4.0" targetplatformversion="3.10.12" detailsurl="https://update.joomla.org/core/sts/extension_sts.xml" />
+	<extension name="Joomla" element="joomla" type="file" version="4.4.0" targetplatformversion="3.10.13" detailsurl="https://update.joomla.org/core/sts/extension_sts.xml" />
+	<extension name="Joomla" element="joomla" type="file" version="4.4.0" targetplatformversion="4.0.4" detailsurl="https://update.joomla.org/core/sts/extension_sts.xml" />
+	<extension name="Joomla" element="joomla" type="file" version="4.4.0" targetplatformversion="4.0.5" detailsurl="https://update.joomla.org/core/sts/extension_sts.xml" />
+	<extension name="Joomla" element="joomla" type="file" version="4.4.0" targetplatformversion="4.0.6" detailsurl="https://update.joomla.org/core/sts/extension_sts.xml" />
 	<extension name="Joomla" element="joomla" type="file" version="4.0.4" targetplatformversion="4.0" detailsurl="https://update.joomla.org/core/sts/extension_sts.xml" />
-	<extension name="Joomla" element="joomla" type="file" version="4.3.4" targetplatformversion="4.1" detailsurl="https://update.joomla.org/core/sts/extension_sts.xml" />
-	<extension name="Joomla" element="joomla" type="file" version="4.3.4" targetplatformversion="4.2" detailsurl="https://update.joomla.org/core/sts/extension_sts.xml" />
-	<extension name="Joomla" element="joomla" type="file" version="4.3.4" targetplatformversion="4.3" detailsurl="https://update.joomla.org/core/sts/extension_sts.xml" />
+	<extension name="Joomla" element="joomla" type="file" version="4.4.0" targetplatformversion="4.1" detailsurl="https://update.joomla.org/core/sts/extension_sts.xml" />
+	<extension name="Joomla" element="joomla" type="file" version="4.4.0" targetplatformversion="4.2" detailsurl="https://update.joomla.org/core/sts/extension_sts.xml" />
+	<extension name="Joomla" element="joomla" type="file" version="4.4.0" targetplatformversion="4.3" detailsurl="https://update.joomla.org/core/sts/extension_sts.xml" />
+	<extension name="Joomla" element="joomla" type="file" version="5.0.0" targetplatformversion="4.4.0" detailsurl="https://update.joomla.org/core/sts/extension_sts.xml" />

```

Comment:
Will the `targetplatformversion="4.4.0"` be updated to the latest 4.4.x with every release, so we allow update only from the latest 4.4.x to 5.y? If that is the case. it is ok as it is. But if we allow updates from any 4.4.x to 5.y, which includes also 4.4.0 stable, then we should change it here and in the extensions_sts.xml to `targetplatformversion="4.4"`. The latter would require that we have the 4.4.1 and later 4.4.x update SQL scripts in the 5.0-dev branch, the former would require to bump that version with every new 4.4.x release.

Reply:
that's one of the questions, I would suggest that you have to be always on the latest 4.4 version before upgrading to 5, that's the reason I set 4.4.0

Reply:
I'm pretty fine with that. But in past it was different and our targetplatformversion did not require that, that's why I had brought it up in one of our maintainers meetings, and there people seemed to favour the other way. Anyway, for now it's ok, and we can see later if we want to change that.

Reply:
On the other hand, in the nightly build XML files this PR allows updating to 5.0 from ANY 4.4 version. We should limit that to 4.4.x, too, with x being the latest 4.4 patch release and being updated with every such patch release.